src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,48 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    this.hidden = false;
+
+    Animal.alive.push(this);
+  }
+
+  checkIfAlive() {
+    if (this.health <= 0) {
+      Animal.alive = Animal.alive.filter((animal) => animal !== this);
+    }
+  }
 }
 
 class Herbivore extends Animal {
   // write your code here
+  // eslint-disable-next-line no-useless-constructor
+  constructor(name, health) {
+    super(name, health);
```

Comment:
[FUNCTION NAMING] - The constructor of the `Herbivore` class should not accept a `health` parameter, as the health is supposed to be `100` by default and not set through the constructor. Please remove the `health` parameter from the constructor.